extensions/todoist/src/components/TaskActions.tsx

```diff
@@ -200,6 +216,15 @@ export default function TaskActions({
 
   return (
     <>
+      <Action
+        title="Complete Task"
+        icon={Icon.Checkmark}
+        shortcut={
+          preferences.completeTaskShortcut === "shiftCmdE" ? { modifiers: ["cmd", "shift"], key: "e" } : undefined
+        }
+        onAction={() => completeTask(task)}
+      />
+
       {isTodoistInstalled ? (
         <Action.Open
           title="Open Task in Todoist"
```

Comment:
```suggestion
          title="Open Task in Todoist"
          shortcut={Keyboard.Shortcut.Common.Open}
```

Since this was the second option before, we lose the backwards compatibility. After this change the option to open task in Todoist will be left without a shortcut. I think it would be better/simpler to just use the same shortcut "Cmd + o" for opening task in the app or browser - rather than different shortcuts that existed before. Please make sure to indicate this breaking change in Changelog.

Reply:
I added `shortcut={Keyboard.Shortcut.Common.Open}` to both the Open in Todoist Browser and Open in Todoist app. (`cmd` + `o`) should still work just like before, only difference is it is no longer the second option to select. `Complete task` is sandwiched in-between "show details" and "Open in Todoist" now.